app/views/providers/interrupt/blocks/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="interruption-panel">
+  <%= page_template(page_title: t(".title_html"), column_width: "full", template: :basic) do %>
+    <%= page_heading(size: "l") %>
+
+    <div class="maximize-text-width">
+      <p class="govuk-body"><%= t(".body_text") %></p>
+      <p class="govuk-body"><%= t(".options") %></p>
+      <ul class="govuk-list govuk-list--bullet">
+        <% t(".option_bullets").each do |bullet| %>
+          <li><%= bullet %></li>
+        <% end %>
+      </ul>
```

Comment:
Similarly, in the latest version of the components, there's [this way of adding lists](https://govuk-components.netlify.app/helpers/list/) which looks like it might be useful for us. Presumably, it would be as simple as something like:
```
<%= govuk_list t(".option_bullets"), type: :bullet %>
```

Non-blocking, but something to consider in the future perhaps?

Reply:
Another good shout for a future PR - main is only on 5.2.1 of govuk.components and the govuk_list was added in 5.2.2